src/datasources/provider-datasource.ts

```diff
@@ -0,0 +1,27 @@
+import { openmrsFetch } from '@openmrs/esm-framework';
+import { BaseOpenMRSDataSource } from './data-source';
+
+export class ProviderDataSource extends BaseOpenMRSDataSource {
+  constructor() {
+    super('/ws/rest/v1/provider?v=custom:(uuid,display)');
```

Comment:
```suggestion
    super(`${restBaseUrl}/provider?v=custom:(uuid,display)`);
```

Reply:
The file has only 1 single usage of the url

Reply:
It's not about the usages. It's a pattern that has happened in all other monorepos to use `restBaseUrl` for the first potion of the rest api url.

Reply:
Done